src/events/request/RequestEventHandler.ts

```diff
@@ -34,15 +34,30 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 		const regex = new RegExp( `(?:${ MentionCommand.ticketLinkRegex.source }|(${ MentionCommand.ticketPattern }))(\\?\\S+)?`, 'g' );
 
 		if ( BotConfig.request.noLinkEmoji && !origin.content.match( regex ) ) {
-			origin.react( BotConfig.request.noLinkEmoji );
-			const timeout = BotConfig.request.noLinkWarningLifetime;
-			const warning = await origin.channel.send( `${ origin.author }, your request doesn't contain any valid ticket reference. If you'd like to add it you can edit your message.` ) as Message;
-			warning.delete( { timeout } );
+			try {
```

Comment:
Should line `L32` in this file be in a `try-catch` block?

Reply:
probably should be, yeah